src/queries/types.ts

```diff
@@ -74,6 +74,8 @@ export const QUERY_TYPES = {
     USER: 'volunteerUser',
     USERS: 'volunteerUsers'
   } as const,
+  VOUCHERS: 'vouchers',
+  VOUCHER: 'voucher',
```

Comment:
we always have singular first, as it also should be the alphabetical order

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/510bdb9e7e17a14badeca2b70989a4ef191a7e92